Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -186,6 +186,13 @@
           "SubId": "Php",
           "ContentsRegex": "$MySqlConnectionStringPhp",
           "MessageArguments": { "secretKind": "PHP MySQL connection string" }
+        },
+        {
+          "Id": "SEC101/114",
```

Comment:
>114 [](start = 24, length = 3)

Please use id 045 for this, not 114

Reply:
Fixed

---
In reply to: [568895718](https://github.com/microsoft/sarif-pattern-matcher/pull/157#discussion_r568895718) [](ancestors = 568895718)